app/models/user.rb

```diff
@@ -12,6 +13,35 @@ class User < ApplicationRecord
     def User.digest(string)
       cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
                                                     BCrypt::Engine.cost
-      BCrypt::Password.create(string, cost: cost)
+                                                    BCrypt::Password.create(string, cost: cost)
+    end
+
+    # ランダムなトークンを返す
+    def User.new_token
```

Comment:
```suggestion
    def self.new_token
```

nits
クラスメソッドは通常このように定義します